### DISCUSSION TITLE
Unable to create new account after upgrade to 6.0.9

### DISCUSSION THREAD

**Discussion by depahk** [firefly-iii/firefly-iii#7483] [Firefly III] [answered]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [X] ... it's [definitely a Firefly III issue, not me](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

I upgraded from 5.6.10 to 6.0.9 some days ago. 
Today I tried to create a new Account but i receive this error:
![Screenshot 2023-05-11 at 00-34-40 Firefly III Exception (](https://github.com/firefly-iii/firefly-iii/assets/44574820/390f3614-5e69-4ad1-abee-916668b07264)

Sorry for the italian language, by the way, this is the relative line from `firefly-iii/storage/logs/ff3-fpm-fcgi-2023-05-11.log` logs:
`[2023-05-11 00:32:07] local.ERROR: Exception is: {"class":"FireflyIII\\Exceptions\\FireflyException","errorMessage":"User has no administration ID.","time":"Thu, 11 May 2023 00:32:07 +0200","file":"\/var\/www\/html\/firefly-iii\/app\/User.php","line":315,"code":0,"version":"6.0.9","url":"https:\/\/firefly.domain.local\/accounts\/store","userAgent":"Mozilla\/5.0 (Windows NT 10.0; Win64; x64; rv:109.0) Gecko\/20100101 Firefox\/112.0","json":true,"method":"POST"}
[2023-05-11 00:32:07] local.WARNING: [Bcc] Could not email or log the error. Please validate your email settings, use the .env.example file as a guide.
[2023-05-11 00:32:07] local.ERROR: User has no administration ID. {"userId":1,"exception":"[object] (FireflyIII\\Exceptions\\FireflyException(code: 0): User has no administration ID. at /var/www/html/firefly-iii/app/User.php:315)
[stacktrace]
#0 /var/www/html/firefly-iii/app/Validation/Administration/ValidatesAdministrationAccess.php(58): FireflyIII\\User->getAdministrationId()
#1 /var/www/html/firefly-iii/app/Http/Requests/AccountFormRequest.php(155): FireflyIII\\Http\\Requests\\AccountFormRequest->validateAdministration()
#2 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Validation/Validator.php(399): FireflyIII\\Http\\Requests\\AccountFormRequest->FireflyIII\\Http\\Requests\\{closure}()
#3 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Validation/Validator.php(448): Illuminate\\Validation\\Validator->Illuminate\\Validation\\{closure}()
#4 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Validation/Validator.php(461): Illuminate\\Validation\\Validator->passes()
#5 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Validation/ValidatesWhenResolvedTrait.php(31): Illuminate\\Validation\\Validator->fails()
#6 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Foundation/Providers/FormRequestServiceProvider.php(30): Illuminate\\Foundation\\Http\\FormRequest->validateResolved()
#7 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Container/Container.php(1304): Illuminate\\Foundation\\Providers\\FormRequestServiceProvider->Illuminate\\Foundation\\Providers\\{closure}()
#8 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Container/Container.php(1268): Illuminate\\Container\\Container->fireCallbackArray()
#9 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Container/Container.php(1254): Illuminate\\Container\\Container->fireAfterResolvingCallbacks()
#10 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Container/Container.php(815): Illuminate\\Container\\Container->fireResolvingCallbacks()
#11 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(933): Illuminate\\Container\\Container->resolve()
#12 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Container/Container.php(731): Illuminate\\Foundation\\Application->resolve()
#13 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(918): Illuminate\\Container\\Container->make()
#14 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Routing/ResolvesRouteDependencies.php(85): Illuminate\\Foundation\\Application->make()
#15 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Routing/ResolvesRouteDependencies.php(50): Illuminate\\Routing\\ControllerDispatcher->transformDependency()
#16 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Routing/ResolvesRouteDependencies.php(29): Illuminate\\Routing\\ControllerDispatcher->resolveMethodDependencies()
#17 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Routing/ControllerDispatcher.php(59): Illuminate\\Routing\\ControllerDispatcher->resolveClassMethodDependencies()
#18 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Routing/ControllerDispatcher.php(40): Illuminate\\Routing\\ControllerDispatcher->resolveParameters()
#19 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Routing/Route.php(259): Illuminate\\Routing\\ControllerDispatcher->dispatch()
#20 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Routing/Route.php(205): Illuminate\\Routing\\Route->runController()
#21 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Routing/Router.php(798): Illuminate\\Routing\\Route->run()
#22 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(141): Illuminate\\Routing\\Router->Illuminate\\Routing\\{closure}()
#23 /var/www/html/firefly-iii/app/Http/Controllers/Account/CreateController.php(71): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#24 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(162): FireflyIII\\Http\\Controllers\\Account\\CreateController->FireflyIII\\Http\\Controllers\\Account\\{closure}()
#25 /var/www/html/firefly-iii/app/Http/Controllers/Controller.php(119): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#26 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(162): FireflyIII\\Http\\Controllers\\Controller->FireflyIII\\Http\\Controllers\\{closure}()
#27 /var/www/html/firefly-iii/app/Http/Middleware/InterestingMessage.php(72): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#28 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): FireflyIII\\Http\\Middleware\\InterestingMessage->handle()
#29 /var/www/html/firefly-iii/app/Http/Middleware/Binder.php(79): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#30 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): FireflyIII\\Http\\Middleware\\Binder->handle()
#31 /var/www/html/firefly-iii/app/Http/Middleware/Range.php(62): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#32 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): FireflyIII\\Http\\Middleware\\Range->handle()
#33 /var/www/html/firefly-iii/vendor/jc5/google2fa-laravel/src/Middleware.php(29): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#34 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): PragmaRX\\Google2FALaravel\\Middleware->handle()
#35 /var/www/html/firefly-iii/app/Http/Middleware/Authenticate.php(75): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#36 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): FireflyIII\\Http\\Middleware\\Authenticate->handle()
#37 /var/www/html/firefly-iii/vendor/laravel/passport/src/Http/Middleware/CreateFreshApiToken.php(50): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#38 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Laravel\\Passport\\Http\\Middleware\\CreateFreshApiToken->handle()
#39 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Session/Middleware/AuthenticateSession.php(60): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#40 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Session\\Middleware\\AuthenticateSession->handle()
#41 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/VerifyCsrfToken.php(78): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#42 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Foundation\\Http\\Middleware\\VerifyCsrfToken->handle()
#43 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/View/Middleware/ShareErrorsFromSession.php(49): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#44 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\View\\Middleware\\ShareErrorsFromSession->handle()
#45 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Session/Middleware/StartSession.php(121): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#46 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Session/Middleware/StartSession.php(64): Illuminate\\Session\\Middleware\\StartSession->handleStatefulRequest()
#47 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Session\\Middleware\\StartSession->handle()
#48 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/AddQueuedCookiesToResponse.php(37): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#49 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Cookie\\Middleware\\AddQueuedCookiesToResponse->handle()
#50 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/EncryptCookies.php(67): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#51 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Cookie\\Middleware\\EncryptCookies->handle()
#52 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(116): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#53 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Routing/Router.php(797): Illuminate\\Pipeline\\Pipeline->then()
#54 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Routing/Router.php(776): Illuminate\\Routing\\Router->runRouteWithinStack()
#55 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Routing/Router.php(740): Illuminate\\Routing\\Router->runRoute()
#56 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Routing/Router.php(729): Illuminate\\Routing\\Router->dispatchToRoute()
#57 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(200): Illuminate\\Routing\\Router->dispatch()
#58 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(141): Illuminate\\Foundation\\Http\\Kernel->Illuminate\\Foundation\\Http\\{closure}()
#59 /var/www/html/firefly-iii/app/Http/Middleware/InstallationId.php(52): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#60 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): FireflyIII\\Http\\Middleware\\InstallationId->handle()
#61 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Http/Middleware/TrustProxies.php(39): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#62 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Http\\Middleware\\TrustProxies->handle()
#63 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TransformsRequest.php(21): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#64 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/ConvertEmptyStringsToNull.php(31): Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest->handle()
#65 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Foundation\\Http\\Middleware\\ConvertEmptyStringsToNull->handle()
#66 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TransformsRequest.php(21): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#67 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TrimStrings.php(40): Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest->handle()
#68 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Foundation\\Http\\Middleware\\TrimStrings->handle()
#69 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/ValidatePostSize.php(27): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#70 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Foundation\\Http\\Middleware\\ValidatePostSize->handle()
#71 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/PreventRequestsDuringMaintenance.php(86): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#72 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Foundation\\Http\\Middleware\\PreventRequestsDuringMaintenance->handle()
#73 /var/www/html/firefly-iii/app/Http/Middleware/SecureHeaders.php(51): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#74 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): FireflyIII\\Http\\Middleware\\SecureHeaders->handle()
#75 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(116): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#76 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(175): Illuminate\\Pipeline\\Pipeline->then()
#77 /var/www/html/firefly-iii/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(144): Illuminate\\Foundation\\Http\\Kernel->sendRequestThroughRouter()
#78 /var/www/html/firefly-iii/public/index.php(73): Illuminate\\Foundation\\Http\\Kernel->handle()
#79 {main}
"}`

I'm not a developer so I don't understand if it is a problem caused by a missing upgrade step that modify some database information. I thought that because I can see inside the db that my 2 users have `NULL` `user_group_id` inside the table `users` and the table `user_groups` is empty.

### Debug information

Debug information generated at 2023-05-11 00:00:00 Europe/Rome for Firefly III version **6.0.9**.

| Scope | Version |
| --- | --- |
| Firefly III | 6.0.9 |
| Firefly III API | 2.0.1 |
| Build | (unknown), (unknown) |
| DB version | 19 (exp. 19) |
| PHP | `8.2.5` |
| Host | `Linux` |

| System info | Value |
| --- | --- |
| System TZ | Europe/Rome |
| Browser TZ | Europe/Rome |
| App environment | local |
| App debug mode | false |
| App cache driver | file |
| App logging | notice, stack |
| Display errors | Off |
| Error reporting | ALL errors |
| Max upload | 2097152 (2 MB) |
| Interface | fpm-fcgi |
| Default language | en_US |
| Default locale | equal |
| BCscale | 12 |
| DB drivers | mysql |
| Current driver | mysql |
| Trusted proxies (.env) | `` |

| User info | Value |
| --- | --- |
| Login provider | eloquent  |
| User guard | web  |
| Headers | N/A, N/A  |
| Stateful domains |  |
| Session start | 2023-05-01 00:00:00 |
| Session end | 2023-05-31 23:59:59 |
| Session first | 2016-01-01 00:00:00 |
| User ID | 1 |
| User language | it_IT |
| User locale | it_IT |
| Attempt at "it_IT.utf8" | 'it_IT.utf8' |
| Attempt at "it_IT.UTF-8" | 'it_IT.UTF-8' |
| User agent | Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:109.0) Gecko/20100101 Firefox/112.0 |

### Expected behaviour

The account creation ;)

### Steps to reproduce

(I hope, update firefly from 5.6.10 to 6.0.9)
Go to https://firefly.domain.local/accounts/create/asset
Set the name and save
Error!

### Additional info

I have a snapshot of the enteire host before the update if needed

**Comment by JC5** [comment]
Hi! Please make sure you run all upgrade commands as instructed in the documentation.

https://docs.firefly-iii.org/firefly-iii/installation/upgrade/

**Reply by depahk** [reply]
I confirm that I followed all instructions. What I missed is to tell that I'm using the self-hosted version and not the docker one (I don't know it really well but it's inside my todo list)

**Reply by JC5** [reply]
I could tell from the debug info :)

I assume this command `php artisan firefly-iii:upgrade-database` had some output, did you notice any errors?

**Reply by depahk** [reply]
At the upgrade time I don't remember, if I run it now (so a second time after the upgrade procedure) the output is that

```
Now seeding the database...
   INFO  Nothing to migrate.

   INFO  Seeding database.

  Database\Seeders\AccountTypeSeeder ................................................................................................. RUNNING
  Database\Seeders\AccountTypeSeeder ............................................................................................ 8.79 ms DONE

  Database\Seeders\TransactionCurrencySeeder ......................................................................................... RUNNING
  Database\Seeders\TransactionCurrencySeeder .................................................................................... 9.35 ms DONE

  Database\Seeders\TransactionTypeSeeder ............................................................................................. RUNNING
  Database\Seeders\TransactionTypeSeeder ........................................................................................ 2.66 ms DONE

  Database\Seeders\PermissionSeeder .................................................................................................. RUNNING
  Database\Seeders\PermissionSeeder ............................................................................................. 1.12 ms DONE

  Database\Seeders\LinkTypeSeeder .................................................................................................... RUNNING
  Database\Seeders\LinkTypeSeeder ............................................................................................... 2.40 ms DONE

  Database\Seeders\ConfigSeeder ...................................................................................................... RUNNING
[2023-05-11 09:15:23] local.WARNING: Database entry exists. Update to latest version (19)
  Database\Seeders\ConfigSeeder ................................................................................................. 2.90 ms DONE

  Database\Seeders\UserRoleSeeder .................................................................................................... RUNNING
  Database\Seeders\UserRoleSeeder ............................................................................................... 3.64 ms DONE

  Database\Seeders\ExchangeRateSeeder ................................................................................................ RUNNING
  Database\Seeders\ExchangeRateSeeder .......................................................................................... 36.00 ms DONE

Fix PostgreSQL sequences.
Command executed successfully.
Now decrypting the database (if necessary)...
Going to decrypt the database.
No decryption required for table "accounts".
No decryption required for table "attachments".
No decryption required for table "bills".
No decryption required for table "budgets".
No decryption required for table "categories".
No decryption required for table "piggy_banks".
No decryption required for table "preferences".
No decryption required for table "tags".
No decryption required for table "transaction_journals".
No decryption required for table "transactions".
No decryption required for table "journal_links".
Done!
Done!
Now executing firefly-iii:transaction-identifiers
This command has already been executed.
Now executing firefly-iii:migrate-to-groups
Database already seems to be migrated.
Now executing firefly-iii:account-currencies
This command has already been executed.
Now executing firefly-iii:transfer-currencies
This command has already been executed.
Now executing firefly-iii:other-currencies
This command has already been executed.
Now executing firefly-iii:migrate-notes
This command has already been executed.
Now executing firefly-iii:migrate-attachments
This command has already been executed.
Now executing firefly-iii:bills-to-rules
This command has already been executed.
Now executing firefly-iii:bl-currency
This command has already been executed.
Now executing firefly-iii:cc-liabilities
No incorrectly stored credit card liabilities.
Now executing firefly-iii:back-to-journals
This command has already been executed.
Now executing firefly-iii:rename-account-meta
This command has already been executed.
Now executing firefly-iii:migrate-recurrence-meta
This command has already been executed.
Now executing firefly-iii:migrate-tag-locations
This command has already been executed.
Now executing firefly-iii:migrate-recurrence-type
This command has already been executed.
Now executing firefly-iii:upgrade-liabilities
This command has already been executed.
Now executing firefly-iii:liabilities-600
This command has already been executed.
Now executing firefly-iii:budget-limit-periods
This command has already been executed.
```

If needed I can restore the VM snapshot and retry the upgrade (I already did it but I can do it again to look better for the errors)

**Reply by JC5** [reply]
No that's not necessary, let me check if the documentation is correct. There's a command to do all of this "in one go" and I thought that was it. Perhaps I forgot to update the docs!

**Reply by JC5** [reply, marked as answer]
Try this set:

```bash
php artisan firefly-iii:upgrade-database
php artisan firefly-iii:correct-database
php artisan firefly-iii:report-integrity
```

**Reply by depahk** [reply]
That's it! I was sure of it during the output of command because I saw that during the second and third command because they output respectively

```
[..]
User "redacted@gmail.com" has no group.
[..]
```

```
[..]
Moved 33 Account objects to the correct group.
Moved 1 Bill objects to the correct group.
Moved 18 Category objects to the correct group.
[..]
```

After these command I have only a cache issue `file_put_contents(/var/www/html/firefly-iii/storage/framework/cache/data/3e/45/3e457a6b1045fdd2bbea7a27a3afb53195ac0089): Failed to open stream: Permission denied`  that I fixed by using  `php artisan cache:clear`

Really thanks for your work! In a few days I'll be in condition to support your work which has been helping me for so many years!

**Reply by JC5** [reply]
Much appreciated, glad I could help! 🎉
